io.github.bkbilly.lnxlink.yaml

```diff
@@ -0,0 +1,165 @@
+id: io.github.bkbilly.lnxlink
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: lnxlink
+finish-args:
+  # Wayland access
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  # For DBus
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --system-talk-name=org.freedesktop.login1
+  - --system-talk-name=org.freedesktop.UPower
```

Comment:
Why are these needed?

Reply:
The Network Manager is to get information about the WiFi: https://github.com/bkbilly/lnxlink/blob/master/lnxlink/modules/wifi.py
The login1 is used to shutdown/restart the computer: https://github.com/bkbilly/lnxlink/blob/master/lnxlink/modules/shutdown.py
The UPower is used for getting the battery information of connected devices: https://github.com/bkbilly/lnxlink/blob/master/lnxlink/modules/battery.py